apps/frontend/src/pages/servers/manage/[id].vue

```diff
@@ -275,6 +277,19 @@ const reconnectInterval = ref<ReturnType<typeof setInterval> | null>(null);
 const isFirstMount = ref(true);
 const isMounted = ref(true);
 
+const INTERCOM_APP_ID = ref("ykeritl9");
```

Comment:
Maybe this should be a env or something incase this has to change for some reason?